LibreNMS/Util/FileCategorizer.php

```diff
@@ -99,6 +102,51 @@ private function validateOs($os)
         return file_exists("includes/definitions/$os.yaml") ? $os : null;
     }
 
+    private function osFromMibs(): void
+    {
+        $mibs = [];
+
+        foreach ($this->items as $file) {
+            if (Str::startsWith($file, 'mibs/')) {
+                $mibs[] = basename($file, '.mib');
+            }
+        }
+
+        $mibs = implode("\n", $mibs);
```

Comment:
nitpick: This line would make more sense in the grep command

Reply:
Fixed